packages/@okta/vuepress-theme-prose/components/LiveWidget.vue

```diff
@@ -0,0 +1,239 @@
+
+          if(areCorrectParanthesis){return 'isValidJsonString'}
+          if (!moreThanOneSymbol) {return 'isBracket'}
+          if (isArr) {return 'isArray'}
+          if (areWrongParanthesis && moreThanOneSymbol) {return 'wrongParenthasis'} 
+          if (!areWrongParanthesis && !isArr && moreThanOneSymbol) {return 'noParenthasis'}
+        }
+        //transform string due to string type
+        const transformByStrType = {
+          isArray: (str) => {
+            const strToArrConversion = str.substring(1,str.length-1)
+                                          .split(', ')
+                                          .map( (el)=> transformByStrType.wrongParenthasis(el))
+                                          .join(', ')
+            return `[${strToArrConversion}]`
+          },
+          isBracket: (str) => str,
+          isValidJsonString: (str)=> str,
+          wrongParenthasis: (str) => `"${str.substring(1, str.length-1)}"`,
+          noParenthasis: (str) => `"${str}"`
+        }
+
+        //add , or : to needed places s
+        const addRightSeparator = (indexStr, str, isLastElement = false)=>{
+          if (str === '{') return str
+          return indexStr === 0 ? `${str}:` : `${str},`
+        }
+
+        const resultNormalized = []
+
+        //clean up initial string from comments, name of object and unneccary semicolon
+        const result = dirtyJSON.slice(dirtyJSON.indexOf('{')).split(/\n/).map( el => el.trim())
+          .filter(el=>
+           !el.startsWith('//') && el!=='')  
+          .map(
+            el => el.includes(' /') ? el.slice(0, el.search(' /')).trim() : el )
+          .map(
+            el => el[el.length-1] == ',' ? el.substring(0, el.length - 1) : el
+          )
+
+        //make valid json string from strings included  
+        result.forEach( (element,index) => {
+          if (element.includes(": ")){
+           element.split(': ').forEach(
+             (elementToTransform, innerIndex) => {
+               const transformed = transformByStrType[defineStrType(elementToTransform)](elementToTransform)
+               resultNormalized.push(addRightSeparator(innerIndex, transformed))
+             }
+           )
+          } else {
+            resultNormalized.push(
+              (element=== '}' && index!= result.length - 1) ?
+               `${element},`: element)
+          }
+        });
+        const resultCleared = resultNormalized.map(
+          (element, index) => {
+            if (
+              resultNormalized[index+1] 
+              &&
+              resultNormalized[index+1].includes('}') 
+              &&
+              element[element.length - 1]===','){
+             return element.substring(0, element.length - 1)
+            } else return element
+          }
+        ).join('').slice(0, -1)
+        return JSON.parse(resultCleared)
+      }, 
+      onJSCodeChange(e){
+        this.configJS = e
+      },
+      onSCSSCodeChange(e){
```

Comment:
It looks like the parameter is not an `Event` object as the `e` tends to represent but actually just the string literal value of the CM object? Can we reflect that better in the variable name?